deploy/docker-compose/sample-workflows/spleen-seg.json

```diff
@@ -0,0 +1,73 @@
+{
+	"name": "ai-spleen-seg",
+	"version": "1.0.0",
+	"description": "AI Spleen Segmentation",
+	"informatics_gateway": {
+		"ae_title": "MONAI-DEPLOY",
```

Comment:
This one uses the same AET as the Liver Seg. So when you have both workflow installed, both will get triggered (both of them are CT series). Also, in both the MAPs, series selection rules are the same and simple too, CT, Primary/Original, MONOCHROME2. Have MAP specific called AET would be better, otherwise need to ask user not to have both workflow def at the same time.

Reply:
Are there any DICOM tags available to differentiate these 2 datasets?

Reply:
There are, but not in attributes that can be relied on. I am including the de-id'ed production CT series from TCIA and it does not even have Body Part attribute.
Use only one model/MAP if you are concerned about setting more AETs.

Reply:
if there are DICOM tags that can be used I can configure the workflow definitions to launch only matching MAPs

Reply:
You were given the DICOM inputs, so can see what's needed to distinguish the two inputs; It is very doable for these couple studies. As I stated before, the WM does not have sufficient capability to filter/select series more generically.

Reply:
Updated the workflow definitions and README with a note of what happens when all definitions are registered.